ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -72,7 +72,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_ipa_ipa_migration.py::TestIPAMigrateScenario1::test_ipa_migrate_prod_mode_hbac_rule test_integration/test_ipa_ipa_migration.py::TestIPAMigrateScenario1::test_ipa_migrate_prod_mode_sudo_rule test_integration/test_ipa_ipa_migration.py::TestIPAMigrateScenario1::test_ipa_migrate_stage_mode_user_uid test_integration/test_ipa_ipa_migration.py::TestIPAMigrateScenario1::test_ipa_migrate_prod_mode_user_uid test_integration/test_ipa_ipa_migration.py::TestIPAMigrateScenario1::test_ipa_migrate_subids
         template: *ci-master-latest
```

Comment:
The tests failed because prepare_ipa_server() wasn't called so there was no "local" server on replicas[0].